api/v1alpha1/api.go

```diff
@@ -378,6 +383,9 @@ type AIServiceBackendSpec struct {
 
 	// Timeouts defines the timeouts that can be configured for an HTTP request.
 	//
+	// Deprecated: Use the `BackendTrafficPolicySpec` for a backend-specific timeout configuration, or
```

Comment:
Maybe worth to add explanations in `AIGatewayRouteRule.Timeouts` about the priorities between different timeouts.

Reply:
https://github.com/envoyproxy/ai-gateway/pull/606/commits/f6d1d981ea20b1e368c1f98ebe8150552227783b done